lib/deriving_Dump.ml

```diff
@@ -252,5 +253,5 @@ module Dump_via_marshal (P : sig type a end) = Defaults (
       let header = readn Marshal.header_size in
       let datasize = Marshal.data_size header 0 in
       let datapart = readn datasize in
-        Marshal.from_string (header ^ datapart) 0
```

Comment:
Could you use ```Marshal.from_bytes``` instead ? Then you wouldn't need to do two conversions to string.

Reply:
Good point! Thanks.

Reply:
Oh crap, I didn't realized deriving should probably be available on OCaml < 4.02. But I'm not sure tho. @vasilisp could you confirm ?

Reply:
Sorry for the late reply. I don't see any need for maintaining compatibility with `< 4.02` or even `4.02.x`.